run-bahmni.sh

```diff
@@ -45,32 +45,40 @@ function checkIfDirectoryIsCorrect {
 }
 
 function start {
-    echo "Executing command: 'docker compose up -d'"
-    echo "Starting Bahmni with default profile from .env file"
-    docker compose up -d
+    if [ "$1" == ".env" ]; then
```

Comment:
No. I don't like this. People should not need to pass `.env` -- that is the default for docker. People should ONLY pass a different .env if they want something besides default. If they pass a param, then that is used for starting the environment, if they don't it remains as `.env` by default.